### DISCUSSION TITLE
Доступы к репозиториям PRO и Service

### DISCUSSION THREAD

**Discussion by BerneRoman** [totumonline/totum-ru-issues-and-discussions#591] [Как это сделать?] [answered]
Доброго дня! На сайте документации битые ссылки в разделе https://ru.docs.totum.online/srv_connection
- https://github.com/totumonline/totum-services
- https://github.com/totumonline/totum-pro
 
Нам нужно установить сервис на локальную машину, но на первом этапе получили затык))

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Она не битая — это приватный репо. Вы должны быть залогинены на GH под пользователем у которого есть доступ.

Приглашение к PRO-репозиторию отправляем на аккаунты зарегистрировавшиеся на https://license.totum-online.ru и указавшие там свой логин на GitHub, или поставившие звезду основному репо: https://github.com/totumonline/totum-mit

Доступ к Сервисам — только для аккаунтов имеющих активную платную лицензию (указать пользователя GH можно на сайте лицензий https://license.totum-online.ru).
